clock/config/database.yml

```diff
@@ -81,5 +81,5 @@ test:
 production:
   <<: *default
   database: clock_production
-  username: peter
-  password: 1007
+  username: pguser
```

Comment:
take a look at this comment, follow the [link](https://github.com/Brightcoders-Bootcamps/kata-07-el-reloj-checador-TM/pull/9#discussion_r515346084) to see it